skbio/alignment/_tabular_msa.py

```diff
@@ -715,8 +792,92 @@ def reindex(self, key=None, keys=None):
             keys_.flags.writeable = False
 
         self._keys = keys_
+        self._minter = minter
 
     @experimental(as_of='0.4.0-dev')
+    def append(self, sequence, minter=None):
```

Comment:
Given my latest comment [here](https://github.com/biocore/scikit-bio/issues/1125#issuecomment-146356126), this method should now accept a minter to apply to `sequence` and a `key` parameter specifying a key for `sequence`. These options should be mutually exclusive. If keys don't exist on the MSA, an error should be thrown if `minter` or `key` is provided.

Much simpler behavior indeed! :shipit: